### ISSUE TITLE
Unixcoder的mask_attention矩阵

### ISSUE THREAD

**Issue by MrBlack0220** [microsoft/CodeBERT#206]
请问如何理解unxicoder.py中的UniXcoder类forward函数中的mask_attention矩阵：
mask = source_ids.ne(self.config.pad_token_id)
attention_mask = mask.unsqueeze(1) * mask.unsqueeze(2)

为什么mask_attention不是source_ids.ne(self.config.pad_token_id)呢？
感谢您的回复！

**Comment by guoday** [comment]
可以是source_ids.ne(self.config.pad_token_id)，效果一样
